commitlint.config.ts

```diff
@@ -167,6 +173,34 @@ module.exports = {
                     ]
                 },
 
+                'footer-correct-place': ({body}: {body:any}) => {
+                    let offence = false;
+
+                    if (body !== null) {
+                        let bodyStr = convertAnyToString(body, "body");
+
+                        let seenBody = false;
+                        let seenFooter = false;
+                        let lines = bodyStr.split(/\r?\n/);
+                        for (let line of lines) {
+                            if (isEmptyLine(line)){
```

Comment:
@parhamsaremi do we really need this check? let's think about it, would an empty line return true for the function isFooterNote? I guess not, so I don't think we need this extra code (maybe I'm wrong, please double check)

Reply:
Yes. an empty line won't be considered as footer but !isFooterNote will be true which is not the behavior I want so I skipped empty lines :thinking: (I also commented out this check and one test failed)

Reply:
ah fair enough, my bad

Reply:
but creating a function for checking if .length is zero looks like overengineering to me :)

Reply:
you're not using that function anywhere else... just check if length is zero right there

Reply:
hmmm sure I'll fix it right now.